social_marketing/overview/livechat.rst

```diff
@@ -0,0 +1,60 @@
+===================================================
+Interact with Customers and Visitors with Live Chat
+===================================================
+
+One of the ways you can build stronger relationships with yours customers is making yourself
+available through live chat. Live chats can have a big impact not just on your customer service,
+but also on sales performance, as it gives users a chance of having real-time, fast and effective
+interaction.
+
+
+Configuration
+=============
+
+Go to :menuselection:`Website --> Configuration --> Settings --> Live Chat` and for the needed
+channel, set the configurations.
+
+.. image:: media/enable_livechat.png
+   :align: center
+   :alt: Enable live chat under website configurations for Odoo Social Marketing
+
+Under *Channel Rules*, choose when the chat pop-up will appear, and an action for a given URL and/or
```

Comment:
This is a general comment, that implies multiple changes in all pages:

Documentation is written in the present tense. The apps already exist, and they already behave in the way described in the documentation.

The future is kept for things that will really appear in the future.
"The dev team is currently working on this feature. It will be available soon!"